lib/Storage/Archiver/XmlArchiver.php

```diff
@@ -120,6 +120,9 @@ public function restore(OutputInterface $output)
             $document = new Document();
             $document->load($file->getPathname());
             $collection = $this->xmlDecoder->decode($document);
+            foreach ($collection as $suite) {
+                $suite->generateUuid();
+            }
```

Comment:
It just mens that we regenerate the UUID everytime, it should be exactly the same - so no harm done, and could be useful in any future UUID migrations - but this was basically just a local hack, not an intended feature.